### DISCUSSION TITLE
Have a 'favourite subreddits' feed section in either main page tabs or bottom navigation bar

### DISCUSSION THREAD

**Discussion by kvithayathil** [Docile-Alligator/Infinity-For-Reddit#1054] [Duplicate Ideas] [answered]
Hi, 

Really liking the app so far! I think it would be great to have the ability to have either a custom tab or custom bottom bar button to display a feed of just the favourited subreddits (if that feature doesn't exist already). I couldn't see that option in the following menus:

<img src="https://user-images.githubusercontent.com/1056073/187827874-6b8be073-fd35-4137-aae9-2d1f9247b6fd.png" width="360">

<img src="https://user-images.githubusercontent.com/1056073/187827879-69ed78b0-4077-401d-b6a6-79fc9bdd956e.png" width="360">

**Comment by Wladefant** [comment, marked as answer]
In the subscriptions page you can also see your favorite subscriptions

**Reply by kvithayathil** [reply]
Appreciate the response. To clarify, I would like to a see a feed of subreddit posts that are limited to just favourite subreddits/subscriptions

**Reply by Wladefant** [reply]
Oh sorry

**Comment by kvithayathil** [comment]
Realised I just described the 'multi reddit' feature. Thank you
